app/src/main/java/com/khaithu/a3ctech_android_final_project/view/MovieDetailView.kt

```diff
@@ -0,0 +1,83 @@
+package com.khaithu.a3ctech_android_final_project.view
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+import com.khaithu.a3ctech_android_final_project.R
+import com.khaithu.a3ctech_android_final_project.api.DisposableManager
+import com.khaithu.a3ctech_android_final_project.model.Genre
+import com.khaithu.a3ctech_android_final_project.model.MovieDetail
+import com.khaithu.a3ctech_android_final_project.presenter.GetMovieDetailPresenter
+import com.khaithu.a3ctech_android_final_project.helper.Constant
+import com.khaithu.a3ctech_android_final_project.helper.GlideHelper
+import com.khaithu.a3ctech_android_final_project.view.interfaceView.IMovieDetailView
+import jp.wasabeef.glide.transformations.BlurTransformation
+import kotlinx.android.synthetic.main.activity_movie_detail_view.*
+
+class MovieDetailView : AppCompatActivity(), IMovieDetailView {
+    private val getMovieDetailPresenter: GetMovieDetailPresenter = GetMovieDetailPresenter(this)
```

Comment:
mPresenter is enough
```suggestion
    private val mPresenter: GetMovieDetailPresenter = GetMovieDetailPresenter(this)
```